app/views/applications/show.njk

```diff
@@ -159,49 +142,28 @@
       {% set referenceId = "second" %}
       {% include "_includes/applications/reference.njk" %}
 
-      <h2 class="govuk-heading-m govuk-!-margin-top-8 govuk-!-font-size-27">Work history</h2>
-
-      {% if application.workHistoryAnswer == "yes" %}
-        {% include "_includes/applications/work-history.njk" %}
-      {% elseif application.workHistoryAnswer == "no--in-full-time-education" %}
-
-        {{ govukSummaryList({
-          rows: [{
-            key: {
-              text: "Do you have any work history?"
-            },
-            value: {
-              html: "No, I have always been in full time education"
-            }
-          }]
-          }) }}
-
-      {% else %}
+      <h2 class="govuk-heading-m govuk-!-margin-top-8 govuk-!-margin-bottom-2 govuk-!-font-size-27">Interview needs</h2>
 
-        {{ govukSummaryList({
-          rows: [{
+      {{ govukSummaryList({
+        rows: [
+          {
             key: {
-              text: "Do you have any work history?"
+              text: "Do you have any interview needs?"
             },
             value: {
-              html: "No, I do not have any work history"
+              text: "Yes" if application.interviewNeeds.response else "No"
```

Comment:
If `application.interviewNeeds.response` is either "Yes" or "No", do you need:

```
"Yes" if application.interviewNeeds.response else "No"
```

or just

```
application.interviewNeeds.response
```